samples/dotnet/KernelHttpServer/Program.cs

```diff
@@ -6,13 +6,19 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.SemanticKernel.Memory;
+using Microsoft.SemanticKernel.Skills.Memory.Sqlite;
+using Microsoft.Data.Sqlite;
+using System.Data.Common;
 
 namespace KernelHttpServer;
 
 public static class Program
 {
     public static void Main()
     {
+        string dbName = "SKDataBase.db";
```

Comment:
I'm guessing this file will be stored side by side with assemblies. While developing this could be frustrating, e.g. often bin/Debug and bin/Release can be deleted. It also opens to the risk of conflicts. Also if the file contains personal data that is left over on disk, we need some way of informing the user

Reply:
Well, I don't know of a good way around the problem. 
At best we can tell the user to create the table themselves and supply that to us. I expect people using the sqlite sample as a reference will already have some experience with sqlite and have their own databases. The idea is for them to set up the data base and plug it in here.
I could delete the table after we are done with it, but that might confuse people who expected this to be a persistent memory store reference app.